client/packages/requisitions/src/RnRForms/DetailView/RnRFormLine.tsx

```diff
@@ -251,11 +251,15 @@ export const RnRFormLine = ({
           sx={{ width: '200px', color: textColor }}
           slotProps={{
             input: {
+              tabIndex: -1,
```

Comment:
also wasn't in issue... they don't want to tab to comment?

Reply:
Ohh yeah sorry, I should have mentioned: 
![Screenshot 2025-05-08 at 3 50 08 PM](https://github.com/user-attachments/assets/6fb4be94-ad77-4254-bcdc-1b75106a4965)